src/Startup/Backend/KeyFile.vala

```diff
@@ -166,9 +166,22 @@ public class Startup.Backend.KeyFile : GLib.Object {
     }
 
     public Entity.AppInfo create_app_info () {
+        var key_name = keyfile_get_string (KeyFileDesktop.KEY_NAME).strip ();
+        if (key_name == null || key_name == "") {
+            key_name = keyfile_get_string (KeyFileDesktop.KEY_EXEC).strip ();
+            if (key_name == null || key_name == "") {
+                key_name = _("Command");
```

Comment:
It looks like the default name that gets set for this is:
```suggestion
                key_name = _("Custom Command");
```

Reply:
Done